tests/test_update_cBioPortal_data.cwl.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+unit tests for the add_af.cwl
+"""
+import os
+import unittest
+from tempfile import TemporaryDirectory
+
+# relative imports, from CLI and from parent project
+if __name__ != "__main__":
+    from .tools import load_mutations, run_cwl, write_table
+    from .settings import CWL_DIR
+
+if __name__ == "__main__":
+    from tools import load_mutations, run_cwl, write_table
+    from settings import CWL_DIR
+
+cwl_file = os.path.join(CWL_DIR, 'update_cBioPortal_data.cwl')
+
+class TestUpdate_cBioPortal_dataCWL(unittest.TestCase):
+    def test_add_af(self):
+        """
+        Test Update_cBioPortal_dataCW with tiny dataset
+        """
+        # maf_lines = [
+        #     ['# comment 1'],
+        #     ['# comment 2'],
+        #     ['Hugo_Symbol', 't_depth', 't_alt_count','tcn','lcn','expected_alt_copies','ccf_expected_copies','ccf_expected_copies_lower','ccf_expected_copies_upper'],
+        #     ['SUFU', '100', '75','4','0.127','0.615','0.375'],
+        #     ['GOT1', '100', '1' ,'4','0.127','0.615','0.375'], # need to change the values
+        #     ['SOX9', '100', '0' ,'4','0.127','0.615','0.375'],
+        # ]
+
+def setUp(self):
```

Comment:
if you are using a setup method for the unit test then it should be one of these;

https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

example;

https://github.com/mskcc/dmp2cmo/blob/febf153f06d050334bae661dc51f3b0c8b238a89/dmp2cmo/test_update.py#L46-L47